docs/iidm/model/index.md

```diff
@@ -3,8 +3,12 @@ title: Internal data model
 layout: default
 ---
 
+Powsybl's internal data model describes electrical networks via [a network core model](#network-core-mode) which models the network equipments
+and its main attributes and [network extensions](#network-extensions) which can describe equipments' specific attributes for custom use cases.
 
-The [Network](network.md) modeling is made of two main objects: substations and lines. A substation can have several voltage levels and a line can be of several types (AC lines, DC lines and AC lines shared between two countries).
+# Network core model
+
+The [Network](network.md) core modeling is made of two main objects: substations and lines. A substation can have several voltage levels and a line can be of several types (AC lines, DC lines and AC lines shared between two countries).
```

Comment:
Model or Modeling: I think we should choose and use the same word everywhere.

I think we should replace **line** by **branch**, because in our model Line stands for AC lines.

Reply:
Isn't it that more confusing as **branches** can mean power transformers (between voltage levels in a **same substation**) and AC lines (between voltage levels that *can* be in **different substations**)? Also DC lines are not branches in IIDM modeling.